src/xrpld/app/main/Application.cpp

```diff
@@ -90,6 +90,9 @@
 
 namespace ripple {
 
+static void
+fixConfigPorts(Config& config, Endpoints const& endpoints);
```

Comment:
Naming things is hard, but I think `updateConfigPorts` would be a better name for this function.